.github/workflows/deploy-forest-calibnet.yml

```diff
@@ -1,40 +1,47 @@
-name: Forest Calibnet
-concurrency: ci-${{ github.ref }}
-
 on:
   pull_request:
     branches:
       - main
     paths:
-      - 'terraform/forest-calibnet/**'
-      - 'terraform/modules/filecoin_node/**'
+      - 'tf-managed/modules/forest-calibnet/**'
+      - 'tf-managed/live/environments/prod/applications/forest-calibnet/**'
+    # This needs to be declared explicitly so that the job is actually
+    # run when moved out of draft.
+    types: [opened, synchronize, reopened, ready_for_review]
   push:
     branches:
       - main
     paths:
-      - 'terraform/forest-calibnet/**'
-      - 'terraform/modules/filecoin_node/**'
+      - 'tf-managed/modules/forest-calibnet/**'
+      - 'tf-managed/live/environments/prod/applications/forest-calibnet/**'
   workflow_dispatch:
 
 jobs:
   deploy-forest-calibnet:
-    name: Deploy
+    env:
+      TF_VAR_slack_enable: true
+      TF_VAR_slack_destination_id: ${{ secrets.SLACK_DESTINATION_ID }}
+      TF_VAR_slack_channel_id: ${{ secrets.SLACK_CHANNEL_ID }}
+      TF_VAR_pagerduty_enable: true
+      TF_VAR_pagerduty_destination_id: ${{ secrets.PAGERDUTY_DESTINATION_ID }}
     runs-on: ubuntu-latest
     permissions: write-all
     steps:
       - name: Checkout the code
         uses: actions/checkout@v4
 
-    # Using Custom Composite action in ./composite-action/terraform folder
-      - name: Composite Action for Deploying Terraform Resources
-        uses: ./composite-action/terraform
+      # Using Custom Composite action in ./composite-action/terragrunt folder
+      - name: Composite Action for Deploying Terragrunt Resources
+        uses: ./composite-action/terragrunt
         with:
           do_token: ${{ secrets.DO_TOKEN }}
           aws_access_key_id:  ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
-          working_directory: terraform/forest-calibnet
-          environment: Calibnet
+          r2_access_key: ${{ secrets.R2_ACCESS_KEY }}
+          r2_secret_key: ${{ secrets.R2_SECRET_KEY }}
+          slack_token: ${{ secrets.SLACK_TOKEN }}
```

Comment:
Are those keys used? R2 is for snapshot downloading, Slack token is for Slack notifications.